src/apps/index.test.tsx

```diff
@@ -0,0 +1,56 @@
+import '@testing-library/jest-dom';
+import { screen, waitFor } from '@testing-library/react';
+import { customRender, logoutUser } from 'utils/test-utils';
+import App from '.';
+
+test('Unauthenticated app renders when no user is present', async () => {
+    await customRender(<App />, {
+        user: undefined,
+    });
+
+    await waitFor(() => {
+        expect(
+            screen.getByText(
+                'When running locally you can login with whatever name you want.'
+            )
+        ).toBeInTheDocument();
+    });
```

Comment:
This await is to allow time for the loading spinner to disappear. Not sure why I couldn't get the waiter added to the `customRender` working... but we can add that functionality later.